Assets/Scripts/Unit/Status/Knockback/Effect/KnockbackFrameEffect.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KnockbackFrameEffect : FrameEffect {
+
+	public KnockbackFrameEffect(Action instance) : base(instance) {}
+
+	public override bool CanExecute(SimulatedDisplacement sim, Direction dir, Board board) {
+		if(board.CheckCoord(sim.GetCurrentVector())) {
+			Tile tile = board.GetTile(sim.GetCurrentVector());
+			return tile.tileType == TileType.PLAINS;
+		}
+		return false;
+	}
+
+	public override bool ExecuteEffect(SimulatedDisplacement sim, Direction dir, Board board)
+	{
+		Unit victim = sim.displacement.unit;
+		victim.FaceDirection(Action.GetOppositeDirection(dir));
```

Comment:
These two lines can be removed since your changes handle direction facing after being pushed.